extide/gradle/src/org/netbeans/modules/gradle/api/GradleBaseProject.java

```diff
@@ -52,7 +53,7 @@ public final class GradleBaseProject implements Serializable, ModuleSearchSuppor
     String name;
     String group = "";
     String description;
-    String version;
+    String version = "";
```

Comment:
```null``` version value has a meaning. Gradle says it is ```unspecified```, when it is an empty string it is an empty string.

Reply:
addressed in 2db82c7